### DISCUSSION TITLE
Shorcut to move to next section

### DISCUSSION THREAD

**Discussion by dmlls** [Hanziness/AnotherPomodoro#252] [Ideas & feature requests]
First of all, thanks for this great and good-looking app!

I've been using it for a while and it always feels somehow cumbersome to me that when a section is finished I can pause it with the space bar, but then I have to click on the arrow to move to the next section.

**What do you think about adding a shortcut for moving to the next section? Perhaps Enter would do?**

**Comment by Hanziness** [comment]
Great catch! Thank you very much. I'll raise this into an issue and start thinking about more keyboard-based options :)
